schema/redis.go

```diff
@@ -0,0 +1,21 @@
+package schema
```

Comment:
Can we change the filename as schema/user_visit_count.go (like schema/lock.go)?

Thanks!